### PR TITLE
Feat: Update doc for emea

### PR DESCRIPTION
**Pull request by marcos-albornoz**
Closes VC-26902

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -264,6 +264,16 @@ Venafi secrets engine:
    Success! Data written to: venafi-pki/roles/vaas
    ```
 
+   **Venafi as a Service EU**:
+
+   ```
+   $ vault write venafi-pki/venafi/vaas \
+       url="https://api.eu.venafi.cloud/" \
```
Comment (sbanic-venafi) [inline README.md:271]: Resolve if you are sure we need to use `api.eu.venafi.com` (as opposed to `api.venafi.eu`).

**Review by sbanic-venafi — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ Vault.
 If you are using Venafi as a Service, verify the following:
 
 - The Venafi as a Service REST API at [https://api.venafi.cloud](https://api.venafi.cloud/swagger-ui.html)
-is accessible from the systems where Vault will be running.
+  or https://api.eu.venafi.cloud(if you have an EU account) is accessible from the systems where Vault will be running.
```
Comment (rvelaVenafi) [inline README.md:74]: use the same format for EU url here 
```
[https://api.eu.venafi.cloud](https://api.eu.venafi.cloud/swagger-ui.html))
```

Comment (rvelaVenafi) [inline README.md:74]: Check which url should be used for EU here as well

**Review by rvelaVenafi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (marcos-albornoz) [reply README.md]: It was determined that the `api.venafi.eu` should be used. The doc was updated to use it.

File: README.md (hunk 2 above)
Comment (marcos-albornoz) [reply README.md]: It's done

Comment (marcos-albornoz) [reply README.md]: It's done

**Review by sbanic-venafi — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -264,6 +264,16 @@ Venafi secrets engine:
...
+   **Venafi as a Service EU**:
+
+   ```
+   $ vault write venafi-pki/venafi/vaas \
+       url="https://api.venafi.eu" \
+       apikey="xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" \
+       zone="Business App\\Enterprise CIT"
+   Success! Data written to: venafi-pki/roles/vaas
+   ```
+
```
Comment (luispresuelVenafi) [inline README.md:276]: add a note below like the following:

**Note**: If you don't specify url for the venafi secret, then we will use "api.venafi.cloud" as default.

**Review by luispresuelVenafi — CHANGES_REQUESTED** [review]

**Review by luispresuelVenafi — APPROVED** [review]

**Review by rvelaVenafi — APPROVED** [review]

**Pull request merged**